docker-compose.yml

```diff
@@ -379,6 +378,8 @@ services:
       - _APP_DB_PASS
       - _APP_LOGGING_PROVIDER
       - _APP_LOGGING_CONFIG
+      - _APP_WORKERS_NUM
+      - _APP_QUEUE_NAME
```

Comment:
Shouldn't these be added to app/config/variables.php? or some .env file?

Reply:
We don't want to introduce these on self-hosted yet since these are temporary.

Reply:
![Screenshot 2023-10-19 at 1 31 46 AM](https://github.com/appwrite/appwrite/assets/20852629/678fe2a3-dc52-4205-8abc-bf1d6f92b5e1)
Here, we pass the args ^